js/src/rive.ts

```diff
@@ -1086,11 +1105,14 @@ export class Rive {
     const elapsedTime = (time - this.lastRenderTime) / 1000;
     this.lastRenderTime = time;
 
-
     // Advance non-paused animations by the elapsed number of seconds
-    const activeAnimations = this.animator.animations.filter(a => a.playing);
+    // Also advance any animations that require scrubbing
+    const activeAnimations = this.animator.animations.filter(a => a.playing || a.needsScrub)
```

Comment:
Moves rendering scrubbing into the main draw loop

Reply:
I did have it in the draw loop before but this way is much better, very nice!